ci-operator/step-registry/telcov10n/metal-single-node-spoke-kpis/tests/cpu-util/telcov10n-metal-single-node-spoke-kpis-tests-cpu-util-commands.sh

```
@@ -231,11 +232,20 @@ function run_ansible_playbook {
 
 function setup_test_result_for_component_readiness {
 
+  local xml_files
+  xml_files=( ${test_results_artifacts_append}*.xml )
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)